minifi-nar-bundles/minifi-framework-bundle/minifi-framework-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,56 @@
+minifi-framework-nar
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Apache NiFi
+    The following NOTICE information applies:
+        Apache NiFi
+        Copyright 2014-2016 The Apache Software Foundation
+
+        This product includes software developed at
+        The Apache Software Foundation (http://www.apache.org/).
+
+        This product includes the following work from the Apache Hadoop project:
```

Comment:
I believe all subelements of the NiFi Notice can be removed here as these are all extension level items that are part of the assembly and not the framework